invoke/loader.py

```diff
@@ -27,7 +29,7 @@ def __init__(self, config=None):
             config = Config()
         self.config = config
 
-    def find(self, name):
+    def find(self, name: str) -> Tuple[IO, str, Tuple[str, str, int]]:
```

Comment:
This might be a good time for us to switch to using `importlib` since `imp` goes away next major Python release? Note to self that we want to dig up whatever PRs are filed for that.

Reply:
@bitprophet I think so too. Could we tackle that after this PR?